server/api.go

```diff
@@ -50,8 +50,19 @@ type PRDetails struct {
 	Reviews            []*github.PullRequestReview `json:"reviews"`
 }
 
+// HTTPHandlerFuncWithUser is http.HandleFunc but userID is already exported
 type HTTPHandlerFuncWithUser func(w http.ResponseWriter, r *http.Request, userID string)
 
+// ResponseType indicates type of response returned by api
+type ResponseType string
+
+const (
+	// ResponseTypeJSON indicates that response type is json
+	ResponseTypeJSON ResponseType = "JSON_RESPONSE"
+	// ResponseTypePlain indicates that response type is text plain
+	ResponseTypePlain = "TEXT_RESPONSE"
```

Comment:
Should this be 
```suggestion
	ResponseTypePlain ResponseType = "TEXT_RESPONSE"
```
?